first-deliverable/data.csv

```diff
@@ -0,0 +1,3 @@
+1,first
+2,second
+3,third
```

Comment:
Be careful on where does end of the file is, as if in further deliverables is required to write over it, the writer would probably depending on how you handle this would inject the new row in the same line breaking the structure of your file.

Reply:
thx